ports/rp2/usbd.c

```diff
@@ -24,6 +24,9 @@
  * THE SOFTWARE.
  */
 
+#include "py/runtime.h"
```

Comment:
It's enough to include just `py/mpconfig.h` here.

Reply:
Even mpconfigport.h should be sufficient.

Reply:
Please use `py/mpconfig.h`, that is the top-level file to use.  It includes possibly `MP_CONFIGFILE` instead of `mpconfigport.h` (any direct include of `mpconfigport.h` is likely wrong and needs to be fixed).